infrastructure/modules/xc3/loadbalancer.tf

```
@@ -22,7 +22,7 @@ resource "aws_lb_target_group" "this" {
   target_type = "instance"
 
   health_check {
-    path                = "/"
+    path                = "/login" #fix health check issues,because we're redirecting traffic to /login.
```

Please make sure, that it is redirected in both dev and prod enviornments.
```suggestion
    path                = "/login" #by default dashboard is redirected to the login page.
```

---

Thanks for the feedback. Actually we're not using loadbalancer in dev environment, but I've tested these changes in dev environment as well; which is working fine.